docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,209 @@
+Configuration.getGlobalConfig().set('purgeOnStart', false);
+
+// Get the request queue from the parent process (step 2)
+const requestQueue = await getOrInitQueue(false);
+
+// Configure crawlee to store the worker-specific data in a separate directory (needs to be done AFTER the queue is initialized when running locally) (step 3)
+const config = new Configuration({
+    storageClientOptions: {
+        localDataDirectory: `./storage/worker-${process.env.WORKER_INDEX}`,
+    },
+});
+```
+
+#### Enabling the request locking experiment, and telling the crawler to use the worker configuration
+
+You might have noticed several lines highlighted in the code above. Those show how you can enable the request locking experiment, as well as how you provide the request queue to the crawler.
+You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
+
+You might have also noticed we passed in a second parameter to the constructor of the crawler, the `config` variable we created earlier.
+This is needed to ensure the crawler uses the worker-specific storages for internal states, and that they do not collide with each other.
+
+#### Why do we use `process.send` instead of `Dataset.pushData`?
+
+Since we use child processes, and each worker process has its own storage space, calling `Dataset.pushData` will not work as we want it to work (each worker would just
+push to it's own personal dataset that is considered the "default" one). Instead, we need to send the data back to the parent process, which has the dataset where we want to
+store the data, in a centralized place.
+
+:::tip Why don't we apply the same logic we did to the request queue to the dataset?
+
+This is a very valid question, but it has a simple answer: since each process tracks its own internal state of how a dataset looks like (when we are scrapping locally),
+the worker processes would get out of sync real fast and would either miss or override data. This is why we need to send the data back to the parent process, which has the dataset
+where we want to store the data, in a centralized place.
+
+Depending on your crawler, this might not be an issue! Each use case has its own quirks, but this is something you should keep in mind when building your scraper.
+
+:::
+
+#### Why did we limit the maximum concurrency to `5`?
+
+This question has a two-fold answer:
+
+- we don't want to overload the source website with requests, so we limit the number of concurrent requests to a reasonable number per worker process
+- we don't want to overload the machine that is running the scraper
+
+This circles back to the initial paragraph about if you should parallelize your scraper or not. Remember: scrape responsibly!
+
+## Other questions
+
+#### Couldn't the `initial-scraper` be merged into the `parallel-scraper`?
```

Comment:
Wouldn't it be better to use _ emphasis for questions instead of h4?  Also if I remember correctly it's markdownlint violation to skip heading levels.

Reply:
> Also if I remember correctly it's markdownlint violation to skip heading levels.

I guess we don't use that rule, as the linter is passing. Do you think it's really needed to enforce this? I think there are other places where it's done this way, one reason is that you don't want those things in the current page navigation (the floating one on the right side). Using emphasis would indeed fix it, but it might not be enough of a visual distinction. I'd maybe prefer block quotes.